osbenchmark/worker_coordinator/runner.py

```diff
@@ -2358,3 +2377,116 @@ async def __aexit__(self, exc_type, exc_val, exc_tb):
 
     def __repr__(self, *args, **kwargs):
         return "retryable %s" % repr(self.delegate)
+
+class DeleteMlModel(Runner):
+    @time_func
+    async def __call__(self, opensearch, params):
+        body= {
+            "query": {
+                "match_phrase": {
+                    "name": {
+                        "query": params.get('model-name')
+                    }
+                }
+            },
+            "size": 1000
+        }
+
+        model_ids = set()
+        try:
+            resp = await opensearch.transport.perform_request('POST', '/_plugins/_ml/models/_search', body=body)
+            for item in resp['hits']['hits']:
+                doc = item.get('_source')
+                if doc:
+                    id = doc.get('model_id')
+                    if id:
+                        model_ids.add(id)
+        except:
+            pass # no current model
```

Comment:
Recommend adding a logging statement that there is no current model before swallowing

Reply:
Pylint also suggests that this is a bare except.
```
osbenchmark/worker_coordinator/runner.py:1288:8: W0702: No exception type(s) specified (bare-except)
osbenchmark/worker_coordinator/runner.py:2404:8: W0702: No exception type(s) specified (bare-except)
osbenchmark/worker_coordinator/runner.py:2444:8: W0702: No exception type(s) specified (bare-except)
```